ironfish/src/network/peerNetwork.ts

```diff
@@ -725,6 +729,8 @@ export class PeerNetwork {
       return true
     }
 
+    this.onTransactionSeen.emit(transaction, received)
```

Comment:
It looks like now this will only emit if we don't early-return in this function (so if the mempool accepts the transaction, this won't fire). Should this event/telemetry only be emitting when a transaction is accepted by the mempool? I'm not sure what your intent is for when to log telemetry here.